lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -1,532 +1,796 @@
 module Cerner
   module Resources
-
     R4_ENCOUNTER ||= {
-      "resourceType": "Encounter",
-      "id": "4027918",
+      "resourceType": 'Encounter',
+      "id": '4221604',
       "meta": {
-        "versionId": "2",
-        "lastUpdated": "2018-05-17T20:51:38.000Z"
+        "versionId": '0',
+        "lastUpdated": '2020-06-25T17:39:25.000Z'
       },
       "text": {
-        "status": "generated",
-        "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\"&gt;&lt;p&gt;&lt;b&gt;Encounter&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;: SMART, JOE&lt;/p&gt;&lt;p&gt;&lt;b&gt;Location&lt;/b&gt;: Baseline East Medical Center, BE Hospital, BE CT&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Inpatient&lt;/p&gt;&lt;p&gt;&lt;b&gt;Service Type&lt;/b&gt;: Emergency Medicine&lt;/p&gt;&lt;p&gt;&lt;b&gt;Class&lt;/b&gt;: inpatient encounter&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: In Progress&lt;/p&gt;&lt;p&gt;&lt;b&gt;Period Start Date&lt;/b&gt;: Apr 20, 2016  9:05 P.M. UTC&lt;/p&gt;&lt;p&gt;&lt;b&gt;Reason For Visit&lt;/b&gt;: Abdominal pain&lt;/p&gt;&lt;p&gt;&lt;b&gt;Attending Physician&lt;/b&gt;: Julius, Scott&lt;/p&gt;&lt;/div&gt;"
+        "status": 'generated',
+        "div": '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Encounter</b></p><p><b>Patient</b>: 108347B, Orion</p><p><b>Location</b>: Baseline East Medical Center</p><p><b>Type</b>: FC MED ELIG JUNIT ENCNTR TYPE CD</p><p><b>Class</b>: inpatient encounter</p><p><b>Status</b>: In Progress</p><p><b>Period Start Date</b>: Feb 28, 1970  6:20 P.M. CST</p><p><b>Period End Date</b>: Apr 30, 1970  7:20 P.M. CDT</p><p><b>Reason For Visit</b>: FEVER</p></div>'
```

Comment:
revert changes. also need to keep gt and lt symbols to prevent html from being rendered

Reply:
https://github.com/cerner/fhir.cerner.com/pull/415/commits/346d1c84ec066ff89fd8f5dfdb7fec58712cc176#diff-8915968a455f59526d7e4635365d9792R13